backend/src/controllers/items-controller.ts

```diff
@@ -38,7 +40,7 @@ export const createItem = async (req: Request, res: Response) => {
         description: description,
         pickupLocation: pickupLocation,
         pickupTime: pickupTime,
-        itemOwnerId: itemOwnerExistence._id
+        itemOwnerId: itemOwnerId
```

Comment:
Use `req.token.userId` instead of `itemOwnerId`
Also, remove all lines of code (express-validator code in routes, logic to check if the itemOwnerId exists here, etc.) associated with `itemOwnerId`. `req.token.userId` will replace the `itemOwnerId`